shared-doc/define-standalone-server-attributes.adoc

```diff
@@ -37,6 +37,13 @@ ifeval::["{standalone-server-type}"=="ha"]
 :serverArguments:  -c standalone-ha.xml
 endif::[]
 
+// Start the standalone server with the Eclipse MicroProfile profile.
```

Comment:
minor enhancement, please add microprofile option in header of the file

Reply:
Done.